fabulaws/library/wsgiautoscale/api.py

```diff
@@ -188,24 +186,63 @@ def _setup_env(deployment_tag=None, environment=None, override_servers={}):
     env.setdefault('syslog_server', False)
 
 
+def _read_local_secrets():
+    """
+    Return a dictionary with the secrets from the local secrets file, if any;
+    else returns None.
+    """
+    secrets_files = [
+        'fabsecrets_{environment}.py'.format(environment=env.environment),
+        'fabsecrets.py'
+    ]
+    secrets_file = None
+    for filename in secrets_files:
+        if os.path.exists(filename):
+            secrets_file = filename
+            break
+    else:
+        print("Warning: No secrets file found. Looked for %r" % secrets_files)
+        return None
+    if secrets_file == 'fabsecrets.py':
+        print("Warning: Getting secrets from fabsecrets.py, which is deprecated. Secrets "
```

Comment:
optional, but might be nice to have these warnings print in a different color, if the terminal allows it: http://docs.fabfile.org/en/1.10/api/core/colors.html